src/ics/icalendar.py

```diff
@@ -38,7 +38,7 @@ class Calendar(CalendarAttrs):
 
     NAME = "VCALENDAR"
     DEFAULT_VERSION: ClassVar[str] = "2.0"
-    DEFAULT_PRODID: ClassVar[str] = "ics.py - http://git.io/lLljaA"
+    DEFAULT_PRODID: ClassVar[str] = "ics.py 0.8.0-dev - http://git.io/lLljaA"
```

Comment:
Greetings, friends!

I have a tiny question/suggestion here - I wonder if this field should be dynamic, based on `__version__` as well? To not rework it with every new version of the package...

Reply:
bumpversion will take care of updating this value automatically, just as it does for all [other files](https://github.com/ics-py/ics-py/pull/303/files#diff-9ea6e1e3dde6d4a7e08c7c88eceed69ca745d0d2c779f8f85219b22266efff7f) where the version is hardcoded. We could also only have the version explicitly set in one place and then insert it dynamically at runtime in all other places, but that would require further code and runtime overhead and not really bring any benefit.

Reply:
@N-Coder awesome, thanks for clarifications! Good to know about `bumpversion` 👍